static/stories/sub_select.md

```diff
@@ -0,0 +1,151 @@
+---
+id: 10
+trl: medium
+category: EUCP science and methodologies
+title: Sub-selection from ensemble projections
+author: T. Palmer et al., MetOffice
+thumbnail: "intro_subsampling.png"
+---
+
+:::Chapter{headline="Introduction and aim" image="subsampling_motivations.png"}
+## Why sub-select from ensemble projections
+
+Climate models represent one of the key datasets to explore potential future
+climate impact, vulnerabilities, and risks. However, not all such models are
+equally skillful in capturing the climate processes that drive climate
+variability and change, particularly at regional scales. There is an interest,
+therefore, in assessing the suitability of these models if they are to be used
+to underpin or inform decision-making.​
+
+In many applications of ensemble climate projections such as CMIP6 it is
+impractical to apply realizations from all 30-50 members, and a smaller subset is
+required. Such applications might include climate change impacts studies (for
+example hydrological models to explore flood impacts) or dynamical downscaling
+with regional climate models.
+:::
+
+:::Chapter{headline="Processed based assessment criteria" image="example_assessment.png"}
+## Processed based assessment criteria
+
+Method builds on the approach developed in McSweeney et al.,(2015, 2018)
```

Comment:
Can we add urls for these references?

Reply:
Yes, do you want me to send them?

Reply:
Please :-)

Reply:
https://doi.org/10.1007/s00382-014-2418-8 McSweeney et al 2015 - I can't find a url for 2018 as its a technical report.

Reply:
here is a citation:  Mcsweeney, C. F., Murphy, J., Sexton, D. M. H., Rostron, J. W., Yamazaki, K., & Harris, G. R. (2018). Selection of CMIP5 members to augment a perturbed-parameter ensemble of global realisations of future climate for the UKCP18 scenarios. Hadley Centre Technical Note. 102

Reply:
@tepmo42 great, thanks. I found this [link](https://digital.nmla.metoffice.gov.uk/IO_28f93601-3178-4c48-8621-fb4388ec66a1/) for the technical report. I will add it.

Reply:
added.